### PR TITLE
add fish summary api

### PR DESCRIPTION
**Pull request by anitanh**
- Added fish summary api

### CODE REVIEW

File: api_views/summary.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+class FishSummary(APIView):
+
+    def get(self, request, *args):
+        category = 'category'
+        fish_summary = \
+            FishCollectionRecord.objects.values(
+                category).annotate(total=Count(category))
+
+        response = JSONRenderer().render(fish_summary)
```
Comment (meomancer) [inline api_views/summary.py]: we don't need this line
please try making fish summary directly as json

**Comment by anitanh** [comment]
api return:
screenshot from 2018-10-16 11-39-44

File: api_views/summary.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+class FishSummary(APIView):
+
+    def get(self, request, *args):
+        category = 'category'
+        fish_summary = \
+            FishCollectionRecord.objects.values(
+                category).annotate(total=Count(category))
+
+        response = {}
+        response['category'] = {}
```
Comment (meomancer) [inline api_views/summary.py]: please also use `category` variable in here

File: api_views/summary.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+        for summary in fish_summary:
+            class_name = summary[category]
+            if not class_name:
+                continue
+            if class_name not in response:
+                response['category'][class_name] = 0
```
Comment (meomancer) [inline api_views/summary.py]: same with here

File: api_views/summary.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+            response['category'][class_name] += summary['total']
```
Comment (meomancer) [inline api_views/summary.py]: same with here

File: api_views/summary.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+                category).annotate(total=Count(category))
+
+        response = {}
+        response['category'] = {}
+
+        for summary in fish_summary:
+            class_name = summary[category]
+            if not class_name:
+                continue
+            if class_name not in response:
```
Comment (meomancer) [inline api_views/summary.py]: this is not necessary

File: api_views/summary.py (hunk 3 above)
Comment (meomancer) [inline api_views/summary.py]: this is not necessary

File: api_views/summary.py (hunk 4 above)
Comment (meomancer) [inline api_views/summary.py]: please also use category variable in here

File: api_views/summary.py (hunk 2 above)
Comment (meomancer) [inline api_views/summary.py]: please define this directly in line 17

**Review by dimasciput — APPROVED** [review]

**Pull request merged**